libs/common/src/admin-console/services/policy/default-vnext-policy.service.spec.ts

```diff
@@ -0,0 +1,572 @@
+import { mock, MockProxy } from "jest-mock-extended";
+import { firstValueFrom, of } from "rxjs";
+
+import { FakeStateProvider, mockAccountServiceWith } from "../../../../spec";
+import { FakeSingleUserState } from "../../../../spec/fake-state";
+import {
+  OrganizationUserStatusType,
+  OrganizationUserType,
+  PolicyType,
+} from "../../../admin-console/enums";
+import { PermissionsApi } from "../../../admin-console/models/api/permissions.api";
+import { OrganizationData } from "../../../admin-console/models/data/organization.data";
+import { PolicyData } from "../../../admin-console/models/data/policy.data";
+import { MasterPasswordPolicyOptions } from "../../../admin-console/models/domain/master-password-policy-options";
+import { Organization } from "../../../admin-console/models/domain/organization";
+import { Policy } from "../../../admin-console/models/domain/policy";
+import { ResetPasswordPolicyOptions } from "../../../admin-console/models/domain/reset-password-policy-options";
+import { POLICIES } from "../../../admin-console/services/policy/policy.service";
+import { PolicyId, UserId } from "../../../types/guid";
+import { OrganizationService } from "../../abstractions/organization/organization.service.abstraction";
+
+import { getFirstPolicy, vNextPolicyService } from "./default-vnext-policy.service";
+
+describe("PolicyService", () => {
+  const userId = "userId" as UserId;
+  let stateProvider: FakeStateProvider;
+  let organizationService: MockProxy<OrganizationService>;
+  let singleUserState: FakeSingleUserState<Record<PolicyId, PolicyData>>;
+
+  let policyService: vNextPolicyService;
+
+  beforeEach(() => {
+    const accountService = mockAccountServiceWith(userId);
+    stateProvider = new FakeStateProvider(accountService);
+    organizationService = mock<OrganizationService>();
+    singleUserState = stateProvider.singleUser.getFake(userId, POLICIES);
+
+    const organizations$ = of([
+      // User
+      organization("org1", true, true, OrganizationUserStatusType.Confirmed, false),
+      // Owner
+      organization(
+        "org2",
+        true,
+        true,
+        OrganizationUserStatusType.Confirmed,
+        false,
+        OrganizationUserType.Owner,
+      ),
+      // Does not use policies
+      organization("org3", true, false, OrganizationUserStatusType.Confirmed, false),
+      // Another User
+      organization("org4", true, true, OrganizationUserStatusType.Confirmed, false),
+      // Another User
+      organization("org5", true, true, OrganizationUserStatusType.Confirmed, false),
+      // Can manage policies
+      organization("org6", true, true, OrganizationUserStatusType.Confirmed, true),
+    ]);
+
+    organizationService.organizations$.mockReturnValue(organizations$);
```

Comment:
This could be more specific:

```suggestion
    organizationService.organizations$.calledWith(userId).mockReturnValue(organizations$);
```